projects/memorizer/kernel-memorizer/Dockerfile

```diff
@@ -39,14 +39,14 @@ ENV KERNEL_PGP2_SIGN=https://www.kernel.org/pub/linux/kernel/v4.x/linux-${KERNEL
 COPY keys.asc keys.asc
 
 # Download and verify kernel
-RUN curl -fsSLO ${KERNEL_SHA256_SUMS} && \
+RUN curl -k -fsSLO ${KERNEL_SHA256_SUMS} && \
```

Comment:
Do you know why the `-k` or `--insecure` ("Allow insecure server connections when using SSL") option is needed?